lib/faker/gov/us/ssn/en_us.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Faker.Gov.Us do
```

Comment:
Module name does not match filename, is that intentional? I'd figure module called `Faker.Gov.Us` should be in a file named `lib/faker/gov/us.ex`.